posthog/hogql/functions/mapping.py

```diff
@@ -399,6 +401,7 @@ def compare_types(arg_types: list[ConstantType], sig_arg_types: tuple[ConstantTy
     "_toUInt128": HogQLFunctionMeta("toUInt128", 1, 1),
     "toFloat": HogQLFunctionMeta("accurateCastOrNull", 1, 1, suffix_args=[ast.Constant(value="Float64")]),
     "toDecimal": HogQLFunctionMeta("toDecimal64OrNull", 2, 2),
+    "DATE": HogQLFunctionMeta("toDate", 1, 1),
```

Comment:
I need something that can convert from a string to a `Date` rather than a `Nullable(Date)`. Clickhouse has a `DATE` function so I thought it made sense to call it this, but I'm open to suggestions

Reply:
There's a big project that @oliverb123 is working on to add a type system, so this shouldn't be needed in the future.

For now, I would probably call this something like `_toDateNotNull`